server/server.js

```diff
@@ -9,7 +9,7 @@ var version   = require('./version.js');
 var port = config.get('port') || 5990;
 var host = config.get('host');
 
-logger.info('ripple-rest (v' + version.getPackageVersion() + ')');
+logger.debug('ripple-rest (v' + version.getPackageVersion() + ')');
```

Comment:
These seem like they should be info level: http://stackoverflow.com/questions/6860186/log-info-vs-log-debug